pages/common/curl.md

```diff
@@ -15,11 +15,15 @@
 
 `curl -O -L -C - {{http://example.com/filename}}`
 
-- Send form-encoded data (POST request of type application/x-www-form-urlencoded):
+- Send form-encoded [d]ata (POST request of type application/x-www-form-urlencoded):
```

Comment:
It's not necessary to use brackets to identify options if they're sufficiently obvious. In this case, since there's only one option, I think it's reasonable to expect people to figure out what it means without adding visual noise to the description. But I'm open to consider other opinions.